R/source.R

```diff
@@ -276,10 +276,23 @@ invoke_cargo <- function(toolchain, specific_target, dir, profile,
 
   tty_has_colors <- isTRUE(cli::num_ansi_colors() > 1L)
 
+  # If there's no `cargo` found, show some helpful messages.
+  no_cargo_found <- isTRUE(nchar(Sys.which("cargo")) == 0)
+  if (no_cargo_found) {
+    ui_w("Can't find {.code cargo} on the PATH. Please review your Rust installation and PATH setups.")
+  }
+
   if (identical(.Platform$OS.type, "windows")) {
     # On Windows, PATH to Rust toolchain should be set by the installer
     cargo_envvars <- NULL
   } else {
+    # Whether or not `cargo` is found on the PATH, we append `~/.cargo/bin` because
+    # it's possible that `rustc` is not found while `cargo` is (c.f. https://github.com/extendr/rextendr/pull/166#discussion_r775803072).
+    # So, we use `no_cargo_found` as a condition only for the friendly message.
+    if (no_cargo_found) {
+      ui_i("For now, {.pkg rextendr} will include {.file ${{HOME}}/.cargo/bin} in PATH temporarily. Please consider adding it in {.file ~/.Renviron}.")
```

Comment:
I suggest to leave either 'For now' or 'temporarily', as both of these sentence fragments communicate the same idea. Also, it sounds better if 'Please consider adding it *to*'